### PR TITLE
Initial 3.0 release post

### PR DESCRIPTION
**Pull request by knolleary**

### CODE REVIEW

**Comment by knolleary** [comment]
Still need to add images etc

File: _posts/2022-07-13-version-3-0-released.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+When you hover over a Junction, it expands to show input and output ports. You can
+move the junction by dragging its body, or add a wire by click/dragging on one of
+its ports - just as you do with nodes.
+
+Junctions do not appear as regular nodes in the palette or flow outline, but can be
+added through the quick-add dialog (Ctrl-click in the workspace) or the new Context Menu.
+
+You can also insert a Junction into an existing wire by slicing through it with the
+mouse by holding `Alt-Shift` when dragging across the wire. Note that some OS might
```
Comment (Steve-Mcl) [inline _posts/2022-07-13-version-3-0-released.md:79]:
```suggestion
mouse by holding <kbd>alt-shift</kbd> when dragging across the wire. Note that some OS might
```

File: _posts/2022-07-13-version-3-0-released.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+When searching for things in the editor, a new toolbar in the workspace provides
+options to quickly jump between the search results.
+
+![continuous-search|416x176, 50%](upload://7vvruWEZmJWtfnDgAhn5DtikKHf.png)
+
+We've added a pair of new actions to help navigate the results:
+
+ - `core:search-previous` - shortcut: `shift-f`
+ - `core:search-next` - shortcut: `f`
```
Comment (Steve-Mcl) [inline _posts/2022-07-13-version-3-0-released.md:105]:
```suggestion
 - `core:search-next` - shortcut: <kbd>f</kbd>
 - `core:search-previous` - shortcut: <kbd>shift-f</kbd>
```

**Review by Steve-Mcl — COMMENTED** [review]
My suggestions - please feel free to ignore :)

File: _posts/2022-07-14-version-3-0-released.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
+---
+layout: blog
+title: Version 3.0 released
+author: nick
+---
+
+Node-RED 3.0 is now available to [install](https://npmjs.org/package/node-red). If upgrading, please read the [upgrade instructions](http://nodered.org/docs/getting-started/upgrading.html).
+
+The [Change Log]() has the full list of changes in this release, but here are the highlights.
```
Comment (knolleary) [inline _posts/2022-07-14-version-3-0-released.md:9]:
```suggestion
The [Change Log](https://github.com/node-red/node-red/blob/1b94cc3ac027bfc38a300867a0b0fd635bf273b8/CHANGELOG.md) has the full list of changes in this release, but here are the highlights.
```

**Pull request merged**